tasks/release.rb

```diff
@@ -44,13 +44,21 @@
       raise "Could not insert PRE in #{file}" unless $1
 
       File.open(file, 'w') { |f| f.write ruby }
+
+      if framework == "actioncable"
+        Dir.chdir("#{root}/actioncable") do
+          version = version.gsub(/\./).with_index { |s, i| i === 2 ? '-' : s }
```

Comment:
> Checking for the 3rd . seems brittle, but will prob work fine for our versions.

Nope; we want to be able to do 4-part versions for security releases. Need to check for non-digits, if that's what's troubling npm.